src/pybind/mgr/cephadm/module.py

```diff
@@ -1440,6 +1440,10 @@ def _add_host(self, spec):
 
         :param host: host name
         """
+
+        if spec.hostname in self.inventory:
+            return "Host '{}' already exists, use 'orch host set-addr' or 'orch host label add/rm' to modify host".format(spec.hostname)
```

Comment:
imagine you have 
```
cpeh orch host add myhost 10.0.0.1
cpeh orch host add myhost 10.0.0.2
```

As a user, I'd expect that the second call would win. Othewise we'd end up with a different semantic than `orch apply` and that would be irritating.

Reply:
reworked the pr so it updates existing hosts